openlibrary/templates/site/banner.html

```diff
@@ -0,0 +1,14 @@
+$def with (announcement, storage_key)
+
+$# Takes following parameters
+$# * announcement (str) -- Rendered content that will be displayed in the banner
+$# * storage_key (str) -- identifier for the banner. If the user dismisses the banner, storage_key will be added to the user's preferences to prevent the banner from showing again.
+
+$# Paste next line into the span element to show the blue banner
+$# <strong>Title</strong> $:announcement &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
```

Comment:
```suggestion
```
This announcement is being passed into the template.  Replacing the `$:announcement` with HTML in the below `span` defeats the purpose of this template.